Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -1185,152 +1124,265 @@
+    <RegEx find="([uU])verljiv" replaceWith="$1vjerljiv" />
     <RegEx find="([uU])vj?ek" replaceWith="$1vijek" />
     <RegEx find="([uU])vet(a|e|i|u|om|ima|nu|nima|noj|nom)?\b" replaceWith="$1vjet$2" />
+    <RegEx find="\b([uU])v[oau]\b" replaceWith="$1h" />
     <RegEx find="([uU])zok" replaceWith="$1zrok" />
     <RegEx find="\bvarvar(a|e|i|u|ima|skom|skim|skoj|skima|ski)" replaceWith="barbar$1" />
     <RegEx find="\bVarvar(a|e|i|u|ima)" replaceWith="Barbar$1" />
     <RegEx find="vazduh(a|u|om)?" replaceWith="zrak$1" />
     <RegEx find="Vazduh(a|u|om)?" replaceWith="Zrak$1" />
-    <RegEx find="vazduš(ni|na|nu|no|nom|nog)" replaceWith="zrač$1" />
+    <RegEx find="vazdušn" replaceWith="zračn" />
+    <RegEx find="Vazdušn" replaceWith="Zračn" />
     <RegEx find="vaspitan" replaceWith="obrazovan" />
     <RegEx find="Vaspitan" replaceWith="Obrazovan" />
     <RegEx find="vaspitn" replaceWith="obrazovn" />
     <RegEx find="Vaspitn" replaceWith="Obrazovn" />
     <RegEx find="([vV])eoma" replaceWith="$1rlo" />
-    <RegEx find="\b([vV])er(a|e|i|u|om|ama|nik|nika|niku|nikom|nica|nice|nici|nicu|nicom|nicama|nicima|ski|ska|sko|sku|skog|skom|skim)\b" replaceWith="$1jer$2" />
+    <RegEx find="\b([vV])er(a|e|i|n|u|om|ama|nik|nika|niku|nikom|nica|nice|nici|nicu|nicom|nicama|nicima|ski|ska|sko|sku|skog|skom|skim)\b" replaceWith="$1jer$2" />
     <RegEx find="([vV])erova(o|la|li|le|lo|še|ti)" replaceWith="$1jerova$2" />
-    <RegEx find="([vV])erovat(an|na|ne|ni|no|nom|nog|nu|noj|nim)" replaceWith="$1jerojat$2" />
+    <RegEx find="([vV])erovat(an|n[aeiou]|no[mgj]|nim)" replaceWith="$1jerojat$2" />
     <RegEx find="([vV])eruj(e|u|em|eš|ete|emo|te)?" replaceWith="$1jeruj$2" />
     <RegEx find="verenic(a|e|i|o|u|ima|ama)" replaceWith="zaručnic$1" />
     <RegEx find="verenik(a|e|u)" replaceWith="zaručnik$1" />
     <RegEx find="\b([vV])j?est(i|ima)?\b" replaceWith="$1ijest$2" />
-    <RegEx find="več(a|e|i|u|em|im|ima|oj)?\b" replaceWith="već$1" />
+    <RegEx find="\b([vV])eč(a|e|i|u|em|im|ima|oj)?\b" replaceWith="$1eć$2" />
+    <RegEx find="([vV])e[čć]n" replaceWith="$1ječn" />
     <RegEx find="\b([vV])eštic(a|e|i|u|om|ama)\b" replaceWith="$1ještic$2" />
-    <RegEx find="\b([vV])enčan(a|i|ica|icu|icom|icama|je|ja|ju|jem|jima)?\b" replaceWith="$1jenčan$2" />
-    <RegEx find="veštačk(a|e|i|o|u|og|om|oj|im|ima)" replaceWith="umjetn$1" />
-    <RegEx find="Veštačk(a|e|i|o|u|og|om|oj|im|ima)" replaceWith="Umjetn$1" />
+    <RegEx find="\b([vV])j?en(ac|c[aeiu]|cem)" replaceWith="$1ijen$2" />
+    <RegEx find="\b([vV])enčan(a|i|ica|icu|icom|icama|j[eau]|jem|jima)?\b" replaceWith="$1jenčan$2" />
+    <RegEx find="veštačk" replaceWith="umjetn" />
+    <RegEx find="Veštačk" replaceWith="Umjetn" />
     <RegEx find="([vV])et(ar|ra|ru|rom|rovi|rovito|rovima)" replaceWith="$1jet$2" />
-    <RegEx find="([vV])ide(la|le|li|ti)" replaceWith="$1idje$2" />
+    <RegEx find="([vV])idel" replaceWith="$1idje" />
```

Comment:
I think, the "l" should not be removed?
```
<RegEx find="([vV])idel" replaceWith="$1idjel" />
```
`videla` ==> `vidjela`, not `videla` ==> `vidjea`